context.c

```diff
@@ -100,18 +126,15 @@ static ssize_t iio_snprintf_context_xml(char *ptr, ssize_t len,
 
 	for (i = 0; i < ctx->nb_attrs; i++) {
 		ret = iio_snprintf(ptr, len,
-				   "<context-attribute name=\"%s\" value=\"",
+				   "<context-attribute name=\"%s\" ",
```

Comment:
You could keep it here, no? Then your iio_xml_print_and_sanitized_param wouldn't need a prefix.

Reply:
I could, but I though is better next to where the value is set.
I mean the first printf set's the name and the next the value.